Image Classification/awesome_classifier.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""Image-Classification.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1StOyfUgZVCPmQWyAJCvGsMwtpe69Uky3
+"""
+
+#Loading the necessary libraries
+import os
+import numpy as np
+import pandas as pd
+import torch 
+import glob
+import torch.nn as nn
+from torchvision.transforms import transforms
+from torch.utils.data import DataLoader
+from torch.optim import Adam
+from torch.autograd import Variable
+import torchvision
+import pathlib
+
+device = torch.device("cuda" if torch.cuda.is_available() 
+                                  else "cpu")
+
+"""Data Preprocessing"""
+
+transformer = transforms.Compose([
+                                  transforms.Resize((150,150)),
+                                  transforms.RandomHorizontalFlip(),
+                                  transforms.ToTensor(), #0-255 will be changed to 0-1 and datatype will be changed from numpy to tensor.
+                                  transforms.Normalize([0.5,0.5,0.5],
+                                                       [0.5,0.5,0.5])
+                                ])
+
+from google.colab import drive
```

Comment:
The script is meant for running on local machines. We do not need anything related to Google Colab